### DISCUSSION TITLE
How to Email Id to Database ?

### DISCUSSION THREAD

**Discussion by kirtanchandak** [supabase/supabase#10717] [Questions] [answered]
Hey, I have implemented user authentication using login with Google and Email. I am getting the email ids in the authentication tab in supabase. I have a doubt - Now how to add these Email ids to postgress/sql data base so I perform operations on it. 
 
![supaAuth](https://user-images.githubusercontent.com/92196705/205481332-e8083c68-fecb-49b1-8267-b7a573890d35.png)
 
How to Add Email IDs in the new Column in Database? Here - 
![database](https://user-images.githubusercontent.com/92196705/205481391-7417dede-2813-440b-a79a-9ea96b3d665b.png)

**Comment by JSmithOner** [comment, marked as answer]
you can create triggers, I won't explaon here but you have tutorial you can find on youtube

basically on insertion of a user entry in the auth table you can copy the `id` column to another table, you can use create this triggers with supabase dashboard but this is some generic postgresql process.

**Comment by kirtanchandak** [comment]
@JSmithOner thanks for the solution, my issue got solved!
